alerter.c

```diff
@@ -14,6 +14,7 @@ int networkAlertStub(float celcius) {
 void alertInCelcius(float farenheit) {
     float celcius = (farenheit - 32) * 5 / 9;
     int returnCode = networkAlertStub(celcius);
+    assert(returnCode == 300);

```

Comment:
This assert would fail even after you correct the code? Watch out for false negatives